server/src/user/user.service.ts

```diff
@@ -113,37 +113,50 @@ export class UserService {
     }
   }
 
-  async deleteUser(userId: number, houseId: number) {
+  async deleteUser(userId: number) {
     try {
       const userHouse = await this.prisma.userHouse.findFirst({
-        where: { userId, houseId },
+        where: { userId },
+        include: { house: true },
```

Comment:
You don't have to include house since you're not using it!

Reply:
Oh true! I fixed it.
93062c0489aa735156a493af5269f5cd99c705c7